features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43012/HCIDriver.cpp

```diff
@@ -92,16 +75,30 @@ class HCIDriver : public cordio::CordioHCIDriver {
 
     virtual void do_initialize()
     {
-        output_mode(bt_host_wake_name, 1);
-        output_mode(bt_device_wake_name, 0);
-        output_mode(bt_power_name, 1);
-
+        //output_mode(bt_host_wake_name, 1);
```

Comment:
Could you comment that change ? Is there any reason to keep that code commented ?

Reply:
Removed commented code with the latest update: https://github.com/ARMmbed/mbed-os/pull/9481/commits/8ca1720e7bbc60688ab49da6bd9ebe1735e64557.

Regarding duplication of HCI implementation: the idea is to design this for extensibility from the beginning.
The pin assignment for CYW43012 and CYW4343X HCI drivers just happens to match for the 4 currently supported boards. In the future, as we add more boards, it may diverge and require  different config.
Since CYW43012 has embedded stack mode, as we may add this feature support in future, it may also make the HCIDriver differ from 4343W.

Reply:
@pan-: the team put more thought on HCI driver design. The driver duplication is now addressed by https://github.com/ARMmbed/mbed-os/pull/9481/commits/6a361837323a4058d68a6fea519cb16e1e6a54dc: there is a single driver for CYW4343W and CYW43012 in features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX. Differences in the implementation are to be handled by constructor argument and json parametrization.
Can you resolve this conversation?